packages/core/android/src/test/kotlin/com/datadog/reactnative/DdLogsTest.kt

```diff
@@ -270,4 +280,130 @@ internal class DdLogsTest {
             attributes = mockContext.toHashMap()
         )
     }
+
+    @Test
+    fun `M not forward logs W SDK is not initialized`() {
+        // When
+        whenever(mockDatadog.isInitialized()) doReturn false
+        var newTestedLogs = DdLogs(mockReactContext, mockLogger, mockDatadog)
+        newTestedLogs.debug(fakeMessage, mockContext, mockPromise)
+        newTestedLogs.info(fakeMessage, mockContext, mockPromise)
+        newTestedLogs.warn(fakeMessage, mockContext, mockPromise)
+        newTestedLogs.error(fakeMessage, mockContext, mockPromise)
+        testedLogs.debugWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+        testedLogs.infoWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+        testedLogs.warnWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+        testedLogs.errorWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+
+        // Then
+        verifyZeroInteractions(mockLogger)
+        val throwableCaptor = argumentCaptor<Throwable>()
+        verify(mockPromise, times(8)).reject(throwableCaptor.capture())
+        assertThat(throwableCaptor.firstValue).hasFieldEqualTo("detailMessage", "Log sent before SDK init")
```

Comment:
we should be able to do `assertThat(throwableCaptor.firstValue.message).isEqualTo("Log sent before SDK init")` without using `hasFieldEqualTo`. Does it allow to do it?